R/ResultFlagsIndependent.R

```diff
@@ -570,7 +610,15 @@ QAPPapproved <- function(.data, clean = TRUE, cleanNA = FALSE) {
 #' data without an associated QAPP document is removed from the dataframe.
 #'
 #' @export
+#' 
+#' @examples 
+#' data(Nutrients_Utah)
+#' 
+#' QAPP_URLs_added <- QAPPDocAvailable(Nutrients_Utah)
```

Comment:
Hmm, the default is clean = FALSE, so I believe the first example would simply flag the data that have an associated QAPP URL provided (maybe "URLs_added" isn't the best way to define this example - perhaps "URLs_flagged" makes more sense). The second example where clean = TRUE would remove rows where no QAPP URL is provided.

Reply:
My mistake! I forgot I had changed the default for this one because most data in the WQP will not have an associated QAPP. 

What do you think of below? 

FlagData_MissingQAPPDocURLs
RemoveData_MissingQAPPDocURLs

Reply:
These sound great! Much more descriptive than the names I had. Should I add comments as well for each example? Looking at other function documentations, they typically include comments explaining what the example will do. Let me know!